rudder-agent/debian/rules

```diff
@@ -83,7 +83,8 @@ binary-arch: install
 #	dh_installlogrotate
 #	dh_installmime
 #	dh_python
-	dh_installinit --no-start -R # restart after upgrade (instead of stop, upgrade, start)
+	dh_installinit --update-rcd-params=disable --no-start -R # restart after upgrade (instead of stop, upgrade, start)
+	dh_installinit --name=rudder --no-start -R # restart after upgrade (instead of stop, upgrade, start)
```

Comment:
Yes, since the database migration done by rudder-upgrade will trigger a PostgreSQL restart